packages/frontend/src/app/api/auth/reset-password/page.tsx

```diff
@@ -0,0 +1,46 @@
+import { Metadata } from 'next';
+import Link from 'next/link';
+import { UserAuthForm } from '@/components/user-auth-form';
+import Image from 'next/image';
+import Logo from '@/../public/logo.svg';
+
+export const metadata: Metadata = {
+  title: 'Reset Password',
+  description: 'Reset your password by entering your email below.'
+};
+
+export default function ResetPassword() {
+  return (
+    <div className="reset-container container my-10 flex flex-col items-center justify-center">
+      <div className="reset-card h-auto w-full rounded-lg border bg-white py-14 md:max-w-md">
+        <div className="reset-content mx-auto flex w-full flex-col justify-center space-y-10 sm:w-[350px]">
+          <div className="reset-header flex flex-col space-y-5 text-center">
+            <Image
```

Comment:
nitpicky(non-blocking): Can the image be wrapped as its own component called "logo".